chaos-days/blog/2024-10-14-realistic-benchmarks/index.md

```diff
@@ -0,0 +1,226 @@
+---
+layout: posts
+title:  "Using realist benchmarks to test performance to derive new cluster configurations"
+date:   2024-10-14
+categories: 
+  - chaos_experiment 
+  - performance
+tags:
+  - benchmark
+authors: rodrigo
+---
+
+# Chaos Day Summary
+
+Our first goal is to further improve and make the benchmarks more 
+realistic, by using a process model that covers the average process 
+orchestration use case and a payload that reflects the reality. 
+
+The second goal is to use these benchmarks to derive new minimal viable 
+cluster configuration that can handle at least 40 process instances per 
+second, while maintaining low backpressure and low latency.
+
+The third goal is to scale out minimal viable cluster configuration 
+resources linearly and see if the performance scales accordingly.
+
+**TL;DR;**
+
+We used a realistic process model to benchmark our system and derived a new 
+cluster configuration based on previous requirements. 
+
+When we scale this base configuration linearly we see that the performance
+increases significantly more than linearly, while maintaining low 
+backpressure and low latency.
+
+## Chaos Experiment
+
+### Expected
+
+We do expect that we can find a cluster configuration that can handle at 40 
+tasks second to be significantly reduced in resources in relation to our 
+smaller clusters ([G3-S HA Plan](https://accounts.cloud.dev.ultrawombat.com/consoleadmin/clusterplans/0af08654-28ec-413a-8dcf-c5938e828ddd)) since 
```

Comment:
:x: I don't think users can access this link :sweat_smile: Maybe let's just summarize the relevant parts in a table?